src/js/loadFiles.js

```diff
@@ -244,9 +244,11 @@ function loadFile(e) {
 
   if(files[0]){
     var _F = files[0];
-    TIPRACKS = {'a':[],'b':[]}; // clear tipracks
+
     _FILENAME = _F.name;
 
+    console.log(_FILENAME);
```

Comment:
Shouldn't be, but the front-end is filled with `console.log`'s so my eyes weren't looking for these